magento/magento2
# Add Tracking Parameters to Varnish Configuration

### Description (*)
Add additional tracking parameters to the Varnish configuration, so that the same page will be cached independent of those tracking parameters.

### Related Pull Requests
https://github.com/magento/magento2/pull/38299
https://github.com/magento/magento2/pull/38302

### Fixed Issues (if relevant)
1. Fixes magento/magento2#38298
2. Fixes https://github.com/magento/magento2/issues/35227

### Manual testing scenarios (*)
1. Open the Magento backend
3. Open Stores > Configuration > Advanced > System > Full Page Cache
4. Export the Varnish configuration
5. Make sure the new tracking parameters are included

### Questions or comments
Previous PRs have not been merged and not been updated. Hence, here is another try to get this over the line.

### Contribution checklist (*)
